### PR TITLE
Fix: Action icon neutral theme

### PR DESCRIPTION
**Pull request by thomasdigby**
Any thoughts on the name here? We're going to need a "grey" icon with hover state for `PasswordInput` visibility toggle, Dialog Close buttons, etc. etc.

### CODE REVIEW

**Comment by andoulla** [comment]
> Any thoughts on the name here? We're going to need a "grey" icon with hover state for `PasswordInput` visibility toggle, Dialog Close buttons, etc. etc.

`Unremarkable` or `bland` could do? all jokes aside neutral sort of makes more sense as it pairs well with the rest of the names

File: src/components/dialog/DialogContent.tsx [hunk 1]
```diff
@@ -93,9 +81,15 @@ export const DialogContent: React.FC<DialogContentProps> = ({
   <>
     <StyledDialogOverlay />
     <StyledDialogContent size={size} {...remainingProps}>
-      <StyledDialogClose>
+      <ActionIcon
+        as={Close}
+        css={{ position: 'absolute', right: '$4', top: '$4' }}
+        label="Close dialog"
```
Comment (andoulla) [inline src/components/dialog/DialogContent.tsx:87]: not a big fan of hardcoded values we can't overwrite? Shall we use this as a default but give the option to pass a translation down to it?

**Review by andoulla — APPROVED** [review]

File: src/components/dialog/DialogContent.tsx (hunk 1 above)
Comment (thomasdigby) [reply src/components/dialog/DialogContent.tsx:87]: Ah yes I knew there was something you'd pick up but couldn't remember what it was 😆 Agree! I'll add a prop for this

Comment (andoulla) [reply src/components/dialog/DialogContent.tsx:87]: It is such a silly little thing but it eventually adds app. there is another instance of it too, I just did the first one

**Review by alexandrunemes — APPROVED** [review]

**Pull request merged**